src/test/java/com/wikia/webdriver/elements/mercury/pages/discussions/PostsListPage.java

```diff
@@ -95,13 +96,8 @@ public void waitForPageReloadWith(final String categoryName) {
     try {
       new FluentWait<>(getPost())
           .withTimeout(DiscussionsConstants.TIMEOUT, TimeUnit.SECONDS)
-          .until(new Predicate<Post>() {
-            @Override
-            public boolean apply(@Nullable Post post) {
-              return post.getPosts().stream()
-                  .allMatch(postEntity -> postEntity.findCategory().endsWith(categoryName));
-            }
-          });
+          .until((Predicate<Post>) post -> post.getPosts().stream()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "post" which hides the field declared at line 34. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)